app/bundles/CoreBundle/Views/Default/head.html.php

```diff
@@ -10,7 +10,11 @@
 ?>
 <head>
     <meta charset="UTF-8" />
-    <title><?php echo $view['slots']->get('pageTitle', 'Mautic'); ?></title>
+    <title><?php echo $view['slots']->get('pageTitle', 'Mautic'); ?>
+	<?php if( !empty($view['slots']->get('headerTitle', '')) ): ?>
+		- <?php echo $view['slots']->get('headerTitle', ''); ?>
```

Comment:
Rearrange to Page Title before `Mautic`. Also a pipe may be a better choice, smaller character so more real estate on tab for other characters, e.g. `Calendar | Mautic`